### PR TITLE
Sync up develop with master.

### PR DESCRIPTION
**Pull request by ryanpcmcquen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,3 +57,13 @@ import { Button } from '@sur-la-table/slt-ui';
...
 For more details, head [here](build/README.md).
+
+## Publishing:
+
+After [building](https://github.com/SurLaTable/slt-ui#building) the repo, do:
+
+```
+cd build
+yarn login
+yarn publish
```
Comment (LeonardMeagher2) [inline README.md]: can we add info about the beta tag?
--tag beta

Comment (ryanpcmcquen) [reply README.md]: @LeonardMeagher2, is it just `yarn publish --tag beta`?

Comment (LeonardMeagher2) [reply README.md]: Yeah

Comment (ryanpcmcquen) [reply README.md]: @LeonardMeagher2, I have added that.

**Pull request merged**